public/index.html

```diff
@@ -1,121 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
   <meta http-equiv="Pragma" content="no-cache" />
   <meta http-equiv="Expires" content="0" />
-</head>
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Google+Sans:400,500|Roboto+Mono|Roboto:300,400,500,700">
+  <title>Developer Survey</title>
+  <style>
+    body {
+      height: 100%;
+      margin: 0;
```

Comment:
I know I have said to not add more files. Given the amount of css in these files, can we add a styles file so the questionnaire won't be overwhelmed?

Reply:
It may result in a flash of unstyled html, but sure.